userdocs/profiles.dev/src/css/schema.module.css

```diff
@@ -0,0 +1,50 @@
+table#schema {
```

Comment:
this looks like the same styling as eksctl [docs page](https://eksctl.io/usage/schema/), imo it's not good to have it here since it doesn't match the rest of the site. Also, the eksctl [docs page](https://eksctl.io/usage/schema/) is really bad to read especially on small devices so i honestly don't like the styling there, I like what we have currently 
![Screenshot 2021-07-22 at 11 25 54](https://user-images.githubusercontent.com/10995114/126625854-e7376712-f7fc-4a17-a68a-fc49476eea8e.png)

Reply:
sure i get that eksctl docs one is hard to read, and i am not suggesting that we copy all of it, but i think we need _some_ styling here.

i would at least like to copy the font and text colour elements from eksctl, eg how the types are shaded differently

Reply:
> i would at least like to copy the font and text colour elements from eksctl, eg how the types are shaded differently
not a fan of that font but i guess we can make it bigger to read, but deffo like the changes you are mentioning it will look nice 👌🏻

Reply:
yeh i am not married to the font, just something "code-y" or "yaml-y" and bigger would be nice i think